src/components/Config/Config.tsx

```diff
@@ -19,6 +21,7 @@ function Config() {
   const [completed, setCompleted] = useState(false);
   const [addedConnetionModalDisplay, setAddedConnetionModalDisplay] = useState(false);
   const [network, setNetwork] = useState(Network.none);
+  const redirectPage = sessionStorage.getItem(strategyPathKey);
```

Comment:
This needs a fallback to `''`. If you just log in and go through the signup flow then the key will be empty and the URL below will be `/null` or `/undefined`